### ISSUE TITLE
[Feature Request] Option for window transparency

### ISSUE THREAD

**Issue by dspacejs** [Taiko2k/TauonMusicBox#422]
For the entire application's background colours. This would look pretty cool and nice to have as an option.

**Comment by Taiko2k** [comment]
It is something I have thought about. However on a technical level it would be very hard to implement, largely in part to due to SDL not exposing any way to do this.

**Comment by Solarunit** [comment]
I think in 2020 (almost 2021) its not very practical and cool anymore. Maybe some cool blur effect instead...

**Comment by dspacejs** [comment]
> I think in 2020 (almost 2021) its not very practical and cool anymore. Maybe some cool blur effect instead...

It doesn't matter whether it's popular or not, it still looks good. And yeah a blur with it would make it look nicer

**Comment by Taiko2k** [comment]
> And yeah a blur with it would make it look nicer

I agree with that point, blurred would look nicer. Though I'm not even sure its possible with most compositors. I know its possible with KDE, though I've never seen it done on GNOME.

**Comment by Solarunit** [comment]
> I know its possible with KDE

its looks pretty cool in this baby WOGUE video:
https://www.youtube.com/watch?v=IOlioUWXwYc
 
>though I've never seen it done on GNOME.

another one
https://www.youtube.com/watch?v=AN-cVA_tCR0

**Comment by C0rn3j** [comment]
This is possible after a migration to SDL3.

#1332

**Comment by Taiko2k** [comment]
@C0rn3j This is still subject to implementation. I intend to add a transparent accents option with along with SDL3 support, but I still have an unsolved technical issue with text rendering that's limiting whole window transparency.

**Comment by Taiko2k** [comment]
Fixed in v11.1.

You can select Glass+ option in Tauon. (still need external blur plugin for blur)
